### PR TITLE
chore: add vercel integration docs

### PR DESCRIPTION
**Pull request by Pagebakers**
#### Checklist

- [ ] run `make test`
- [ ] PR title must follow [conventional-commit-standard](https://github.com/wundergraph/wundergraph/blob/main/CONTRIBUTING.md#conventional-commit-standard)
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message and code follows the [Code of conduct](https://github.com/wundergraph/wundergraph/blob/main/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Comment by fiam** [comment]
LGTM

File: docs-website/src/pages/docs/cloud/integrations/vercel.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+description: How to integrate WunderGraph with Vercel
+---
+
+How to integrate WunderGraph with Vercel.
+
+{% callout title="This feature isn't publicly available yet." /%}
+
+## How it works
+
+The WunderGraph Vercel integration allows you sync your WunderGraph project settings with your Vercel projects.
```
Comment (Aenimus) [inline docs-website/src/pages/docs/cloud/integrations/vercel.md:13]: allows you sync -> allows you to sync

File: docs-website/src/pages/docs/cloud/integrations/vercel.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+This allows you to fully automate backend and frontend deployments. When the integration is enabled, WunderGraph will sync the public hostname of your API to your Vercel project.
+
+## Installation
+
+If you haven't imported your repository to Vercel yet, please do so first. You can find more information about that [here](https://vercel.com/docs/git-integrations).
+
+Head over to your Vercel dashboard and click on the `Integrations` tab. Go to `Browse Marketplace` and Search for `WunderGraph` and click on the `Add integration` button.
```
Comment (Aenimus) [inline docs-website/src/pages/docs/cloud/integrations/vercel.md:20]: Go to `Browse Marketplace` and Search for `WunderGraph` and -> Go to Go to `Browse Marketplace`, search for `WunderGraph`, and

File: docs-website/src/pages/docs/cloud/integrations/vercel.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+Select the Vercel Account that you want to add the integration to and click `Continue.`. Choose All Project and select individual project that you want to give WunderGraph access to and continue.
```
Comment (Aenimus) [inline docs-website/src/pages/docs/cloud/integrations/vercel.md:22]: Select the Vercel Account that you want to add the integration to and click -> Select the Vercel Account to which you wish to add integration and click

Comment (Aenimus) [inline docs-website/src/pages/docs/cloud/integrations/vercel.md:22]: Choose All Project -> Choose `All Projects`

File: docs-website/src/pages/docs/cloud/integrations/vercel.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+Now you can verify the permissions that are given to WunderGraph and click `Add integration`, a popup opens that will redirect to WunderGraph.
```
Comment (Aenimus) [inline docs-website/src/pages/docs/cloud/integrations/vercel.md:23]: and click `Add integration`, a popup opens that will redirect to WunderGraph. -> and click `Add Integration`. A popup will open that will redirect to WunderGraph.

File: docs-website/src/pages/docs/cloud/integrations/vercel.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+Log in if you're not logged in yet.
```
Comment (Aenimus) [inline docs-website/src/pages/docs/cloud/integrations/vercel.md:25]: Log in if you're not logged in yet. -> Log in if you're not already logged in.

**Review by Aenimus — CHANGES_REQUESTED** [review]
Just some copywriting

File: docs-website/src/pages/docs/cloud/integrations/vercel.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+## How it works
+
+The WunderGraph Vercel integration allows you to sync your WunderGraph project settings with your Vercel projects.
+This allows you to fully automate backend and frontend deployments. When the integration is enabled, WunderGraph will sync the public hostname of your API to your Vercel project.
+
+## Installation
+
+If you haven't imported your repository to Vercel yet, please do so first. You can find more information about that [here](https://vercel.com/docs/git-integrations).
+
+Head over to your Vercel dashboard and click on the `Integrations` tab. Go to `Browse Marketplace`, Search for `WunderGraph` and click on the `Add integration` button.
```
Comment (Aenimus) [inline docs-website/src/pages/docs/cloud/integrations/vercel.md:20]: Not enough to say "no" but capital s for Search?

Comment (Aenimus) [reply docs-website/src/pages/docs/cloud/integrations/vercel.md:20]: and no Oxford comma ;D

File: docs-website/src/pages/docs/cloud/integrations/vercel.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+
+Select the Vercel Account to which you wish to integrate and click `Continue.`. Choose all Project and select individual project that you want to give WunderGraph access to and click `Continue`.
```
Comment (Aenimus) [inline docs-website/src/pages/docs/cloud/integrations/vercel.md:22]: Should All Project be capitalised, be in backticks and be plural? `All Projects`?

Comment (Aenimus) [reply docs-website/src/pages/docs/cloud/integrations/vercel.md:22]: and select THE individual project to which you want/wish to give WunderGraph access and click `Continue`

File: docs-website/src/pages/docs/cloud/integrations/vercel.md [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+Now you can verify the permissions that are given to WunderGraph and click `Add integration`, a popup will open that will redirect to WunderGraph.
```
Comment (Aenimus) [inline docs-website/src/pages/docs/cloud/integrations/vercel.md:23]: Period and capital A.

File: docs-website/src/pages/docs/cloud/integrations/vercel.md [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+
+Log in if you're not already logged in.
+
+Now you can select the Vercel project that you want to connect to your WunderGraph project and click `Connect Projects`. You will be redirected back to Vercel and the integration is now installed. 🥳
```
Comment (Aenimus) [inline docs-website/src/pages/docs/cloud/integrations/vercel.md:27]: Vercel, and the (comma)

Comment (Pagebakers) [reply docs-website/src/pages/docs/cloud/integrations/vercel.md:27]: , and? either , or and. Or is that only Dutch rules?

**Review by Aenimus — APPROVED** [review]

**Pull request merged**